### PR TITLE
updating spglib to 2.0.2

### PR DESCRIPTION
**Pull request by AndresOrtegaGuerrero**
This PR address #486

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -34,7 +34,7 @@ install_requires =
     pymysql~=0.9
     nglview~=3.0
     optimade-client==2022.9.19
-    spglib~=1.14
+    spglib~=2.0.2
```
Comment (yakutovicha) [inline setup.cfg:37]: I would do `~2.0`. Versions such as `2.1`, `2.2`, and `2.100` should still be ok.

```suggestion
    spglib~=2.0
```

**Review by yakutovicha — CHANGES_REQUESTED** [review]
suggestion

File: setup.cfg (hunk 1 above)
Comment (danielhollas) [reply setup.cfg:37]: Agree, I think in general we should not pin to specific versions, otherwise we'll be running into this issue over and over again.

**Review by danielhollas — COMMENTED** [review]
Just asking the obvious question, since the spglib is used in the viewer, has this been tested?

**Review by yakutovicha — DISMISSED** [review]
LGTM!

**Comment by danielhollas** [comment]
Just a question, does this mean that when this is merged and released, that the new AWB version will be incompatible with old QeApp versions?

If that is the case, I would suggest to relax the constraint here and allow both old and new spglib versions here, since it does not seem to affect the AWB code anyway.

**Comment by AndresOrtegaGuerrero** [comment]
@danielhollas I should do this by just `spglib >=1.14` ?

**Comment by yakutovicha** [comment]
> Just a question, does this mean that when this is merged and released, that the new AWB version will be incompatible with old QeApp versions?

I am confused, why QE app? I checked - QE app doesn't depend on `spglib`.

**Review by yakutovicha — APPROVED** [review]

**Pull request merged**